doc/releases/changelog-dev.md

```diff
@@ -522,46 +477,122 @@
+  The :func:`~.transforms.set_shots` transform can be used as a decorator:
 
-  rs_specs = qml.specs(rs_circuit)(x, y)["resources"]
-  sk_specs = qml.specs(sk_circuit)(x, y)["resources"]
+  ```python
+  @partial(qml.set_shots, shots=2)
+  @qml.qnode(qml.device("default.qubit", wires=1))
+  def circuit():
+      qml.RX(1.23, wires=0)
+      return qml.sample(qml.Z(0))
   ```
 
-  Decomposing with `method="rs"` instead of `method="sk"` gives a significant reduction in overall 
-  gate counts, specifically the `qml.T` count:
+  ```pycon
+  >>> circuit()
+  array([1., -1.])
+  ```
+  
+  Additionally, it can be used in-line to update a circuit's `shots`:
 
   ```pycon
-  >>> print(rs_specs.num_gates, sk_specs.num_gates)
-  267 48637
-  >>> print(rs_specs.gate_types['T'], sk_specs.gate_types['T'])
-  104 8507
+  >>> new_circ = qml.set_shots(circuit, shots=(4, 10)) # shot vector
+  >>> new_circ()
+  (array([-1.,  1., -1.,  1.]), array([ 1.,  1.,  1., -1.,  1.,  1., -1., -1.,  1.,  1.]))
   ```
 
-<h3>Improvements 🛠</h3>
+<h4>QChem</h4>
+
+* The `qchem` module is upgraded with new functions to construct a vibrational Hamiltonian in 
+  the Christiansen representation. 
+  [(#7491)](https://github.com/PennyLaneAI/pennylane/pull/7491)
+  [(#7596)](https://github.com/PennyLaneAI/pennylane/pull/7596)
+
+  The new functions :func:`christiansen_hamiltonian` and :func:`qml.qchem.christiansen_bosonic` can
+  be used to create the qubit and bosonic form of the Christiansen Hamiltonian, respectively. These
+  functions need input parameters that can be easily obtained by using the
+  :func:`christiansen_integrals` and :func:`vibrational_pes` functions. Similarly, a Christiansen
+  dipole operator can be created by using the :func:`christiansen_dipole` and
+  :func:`christiansen_integrals_dipole` functions.
+
+  ```python
+  import pennylane as qml
+  import numpy as np
+
+  symbols  = ['H', 'F']
+  geometry = np.array([[0.0, 0.0, -0.40277116], [0.0, 0.0, 1.40277116]])
+  mol = qml.qchem.Molecule(symbols, geometry)
+  pes = qml.qchem.vibrational_pes(mol, optimize=False)
+  ham = qml.qchem.vibrational.christiansen_hamiltonian(pes, n_states = 4)
+  ```
+
+  ```pycon
+  >>> ham
+  (
+      0.08527499987546708 * I(0)
+    + -0.0051774006335491545 * Z(0)
+    + 0.0009697024705108074 * (X(0) @ X(1))
+    + 0.0009697024705108074 * (Y(0) @ Y(1))
+    + 0.0002321787923591865 * (X(0) @ X(2))
+    + 0.0002321787923591865 * (Y(0) @ Y(2))
+    + 0.0008190498635406456 * (X(0) @ X(3))
+    + 0.0008190498635406456 * (Y(0) @ Y(3))
+    + -0.015699890427524253 * Z(1)
+    + 0.002790002362847834 * (X(1) @ X(2))
+    + 0.002790002362847834 * (Y(1) @ Y(2))
+    + 0.000687929225764568 * (X(1) @ X(3))
+    + 0.000687929225764568 * (Y(1) @ Y(3))
+    + -0.026572392417060237 * Z(2)
+    + 0.005239546276220405 * (X(2) @ X(3))
+    + 0.005239546276220405 * (Y(2) @ Y(3))
+    + -0.037825316397333435 * Z(3)
+  )
+  ```  
+
+<h4>Experimental FTQC module</h4>
```

Comment:
I think there are a few in here that are not FTQC, maybe everything after this one:

> * Adds a new allocation module containing allocate and deallocate instructions for requesting dynamic wires. This is currently experimental and not integrated. https://github.com/PennyLaneAI/pennylane/pull/7704 https://github.com/PennyLaneAI/pennylane/pull/7710

Maybe we can move them to other improvements.